invenio_records_marc21/ui/theme/assets/semantic-ui/js/invenio_records_marc21/deposit/RecordSchema.js

```diff
@@ -0,0 +1,144 @@
+// This file is part of Invenio.
+//
+// Copyright (C) 2021 Graz University of Technology.
+//
+// Invenio-Records-Marc21 is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see LICENSE file for more
+// details.
+
+import _get from "lodash/get";
+import _has from "lodash/has";
+import axios from 'axios';
+
+export class RecordSchema {
```

Comment:
add Marc21 as a "namespace" to RecordSchema -> Marc21RecordSchema. So all classes have the same naming schema